### PR TITLE
Remove _total from prometheus_tsdb_storage_blocks_bytes

### PR DESCRIPTION
**Pull request by brian-brazil**
Signed-off-by: Brian Brazil <brian.brazil@robustperception.io>

### CODE REVIEW

**Comment by krasi-georgiev** [comment]
LGTM

**Comment by brian-brazil** [comment]
Those are counters, so there's no need to change them.

**Comment by brian-brazil** [comment]
Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +12,7 @@
  - [REMOVED] `PrefixMatcher` is considered unused so was removed.
  - [CLEANUP] `Options.WALFlushInterval` is removed as it wasn't used anywhere.
  - [FEATURE] Add new `LiveReader` to WAL pacakge. Added to allow live tailing of a WAL segment, used by Prometheus Remote Write after refactor. The main difference between the new reader and the existing `Reader` is that for `LiveReader` a call to `Next()` that returns false does not mean that there will never be more data to read.
+ - [CHANGE] Empty blocks are not written during compaction [#374](https://github.com/prometheus/tsdb/pull/374)
```
Comment (codesome) [inline CHANGELOG.md]:
```suggestion
```

Comment (krasi-georgiev) [reply CHANGELOG.md]: oo shoot I missed that.

Comment (krasi-georgiev) [reply CHANGELOG.md]: fixed in https://github.com/prometheus/tsdb/pull/507

**Pull request merged**